wetterdienst/enumerations/datetime_format_enumeration.py

```diff
@@ -7,3 +7,8 @@ class DatetimeFormat(Enum):
     YMDHM = "%Y%m%d%H%M"
     YMDH_COLUMN_M = "%Y%m%d%H:%M"
     YMD_TIME_H = "%Y-%m-%dT%H"
+
+    # For RADOLAN file datetime parsing
+    YM = "%Y%m"
+    ymdhm = "%y%m%d%H%M"
```

Comment:
Why is this variable in lower letters?

Reply:
I still have to figure this out but there are two strings that are almost similar here:

"%y%m%d%H%M" and "%Y%m%d%H%M" thats why we need two enumerations that can be distinguished

Reply:
Okay might be a suitable solution. I will take some time to find  generic and pythonic way to handle date format strings.